common-app-lib/src/cmr/common_app/services/kms_lookup.clj

```diff
@@ -32,16 +32,19 @@
    :projects [:short-name :long-name]
    :providers [:short-name :long-name]
    :spatial-keywords [:category :type :subregion-1 :subregion-2 :subregion-3]
-   :concepts [:short-name]})
+   :concepts [:short-name]
+   :iso-topic-categories [:short-name]})
 
 (defn- normalize-for-lookup
   "Takes a map (either a UMM-C keyword or a KMS keyword) and a list of fields from the map which we
   want to use for comparison. We return a map containing only the keys we are interested in and with
   all values in lower case."
   [m fields-to-compare]
-  (->> (select-keys m fields-to-compare)
-       util/remove-nil-keys
-       (util/map-values str/lower-case)))
+  (if (map? m)
+    (->> (select-keys m fields-to-compare)
+         util/remove-nil-keys
+         (util/map-values str/lower-case))
+    {(first fields-to-compare) (str/lower-case m)}))
```

Comment:
It seems like this change should also update the docstring ... since there's now an allowed condition for `m` not being a map ...? It would be good to say when `m` is not expected to be a map what is being returned in that case (in particular, what the returned data structure is confirming to).